### ISSUE TITLE
Local issue running Hospital Readmission notebook

### ISSUE THREAD

**Issue by bricha2** [kohpangwei/influence-release#23]
I have been working on replicating the hospital readmission notebook locally and I am getting output that differs substantially from the expected output. Below, I show side-by-side my output vs the expected output. I think the major issue is that the number of parameters that my model outputs is 122 when it's supposed to be 127. I can't localize where this change is occurring because the dimensions of my input data are as expected: 127 columns. Any guidance you can provide on why this might be occurring would be greatly appreciated!
<img width="379" alt="Screen Shot 2021-06-10 at 7 07 29 PM" src="https://user-images.githubusercontent.com/13242069/121608612-593f0a80-ca20-11eb-8282-25b3699f5e5a.png">
<img width="384" alt="Screen Shot 2021-06-10 at 7 07 53 PM" src="https://user-images.githubusercontent.com/13242069/121608613-593f0a80-ca20-11eb-97a3-8519a39e86f5.png">

**Comment by kohpangwei** [comment]
Hi Brianna,

Hmm, I'm not sure why the output dimensions are changing. If you run the hospital notebook in this repo without any changes (https://github.com/kohpangwei/influence-release/blob/master/scripts/hospital_readmission.ipynb), do you get the expected number of parameters? If you don't, then it could perhaps be due to version changes (since this code is quite old)?

Is your `input_dim` 127?

**Comment by bricha2** [comment]
Hello! Yeah, I did have issues running this repo as is. The docker installation was too outdated for my system, it seems. I got the correct input_dim, but the model still ended up with a different number of parameters (122). Even when I change the dataset (to the adult dataset), I am still getting the same issue where the original dimensions (97) are somehow converted to 122 in the model. I wasn't able to figure out the cause of this, but I assume it was a version issue. Thanks for your time and assistance, anyhow!
